contact/lib/data/datasource/auto_complete_datasource.dart

```diff
@@ -0,0 +1,7 @@
+
+import 'package:contact/contact/model/tmail_contact.dart';
+import 'package:jmap_dart_client/jmap/account_id.dart';
+
+abstract class AutoCompleteDataSource {
```

Comment:
No I mean we should extends https://github.com/linagora/tmail-flutter/blob/master/lib/features/composer/data/datasource/autocomplete_datasource.dart. No need to add more datasource